+++ PKGBUILD
@@ -1,11 +1,11 @@
 # Maintainer: Tom Richards <tom@tomrichards.net>

 _nginxver=1.6.0
-_passengerver=4.0.45
+_passengerver=4.0.46

 pkgname=nginx-passenger
 pkgver=1.6.0
-pkgrel=4
+pkgrel=5
 pkgdesc="HTTP Server with Passenger Module"
 arch=('i686' 'x86_64')
 url='http://nginx.org'
@@ -29,7 +29,7 @@
 	'service'
 	'logrotate')
 sha256sums=('943ad757a1c3e8b3df2d5c4ddacc508861922e36fa10ea6f8e3a348fc9abfc1a'
-            '9889fafdd5524ba8ef26bcc98df69ab7d6b5ab6042a34e8690b8acd03e3ff1ab'
+            '70cb6ca43ad9f9cf934f6069a109720e41b68fd590a61a101396a9035d90e84b'
             'ef8c92df29814a35f133ec5d9fef896f93709068aa3ca964b64aae68bdec2ab6'
             '05fdc0c0483410944b988d7f4beabb00bec4a44a41bd13ebc9b78585da7d3f9b'
             '272907d3213d69dac3bd6024d6d150caa23cb67d4f121e4171f34ba5581f9e98')
